Functions.Templates/Templates-v2/HttpTrigger-Python/template.json

```diff
@@ -8,10 +8,10 @@
         {
             "name": "Create New Project with HttpTrigger function",
             "type": "CreateNewApp",
-            "input" : [
+            "inputs" : [
                 {
                     "assignTo": "$(APP_FILENAME)",

```

Comment:
Also if the value of this variable is function_app.py, can it be saved as a global variable?
We would need to access this filename while adding blueprints. When a blueprint is added, we need to register the blueprint in the APP_FILENAME file. Since in the adding blueprint job, there is not input to read the APP_FILENAME, we need to way to preserve the state if this variable so that it can be used later.

Reply:
Discussed various strategies to approach this offline.